rb/spec/integration/selenium/webdriver/element_spec.rb

```diff
@@ -145,39 +145,41 @@ module WebDriver
         end
       end
 
-      context 'size and location' do
-        it 'should get current location' do
-          driver.navigate.to url_for('xhtmlTest.html')
-          loc = driver.find_element(class: 'header').location
-
-          expect(loc.x).to be >= 1
-          expect(loc.y).to be >= 1
-        end
+      not_compliant_on browser: :ie do
```

Comment:
#4480 - Jim thinks it has to do with a current issue in the atoms. It'll probably get fixed quickly enough.